libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -1136,6 +1182,35 @@ - (SFUserAccount*)createUserAccount:(SFOAuthCredentials *)credentials {
     [self saveAccountForUser:newAcct error:nil];
     return newAcct;
 }
+- (void)createNativeUserAccount:(NSData *)data {
+    if (![NSThread isMainThread]) {
+        dispatch_async(dispatch_get_main_queue(), ^{
+            [self createNativeUserAccount:data];
+        });
+        return;
+    }
+    
+    UIScene *scene = SalesforceSDKManager.sharedManager.nativeLoginViewController.view.window.windowScene;
+    SFSDKAuthSession *authSession = self.authSessions[scene.session.persistentIdentifier];
+    
+    // Dummy request is necessary for flow
+    SFSDKOAuthTokenEndpointRequest *request = [[SFSDKOAuthTokenEndpointRequest alloc] init];
+    request.additionalOAuthParameterKeys = [NSArray new];
+    
+    // Let SFSDKOAuth2 do error handling
+    [(SFSDKOAuth2 *)self.authClient() handleTokenEndpointResponse:^(SFSDKOAuthTokenEndpointResponse *response) {
```

Comment:
could `handleTokenEndpointResponse` be added to `SFSDKOAuthProtocol` instead of casting?

Reply:
Perhaps, but that would make it public.  I was trying to make as little private API public as possible.